### PR TITLE
Fix #964 - Rewrite header use position:sticky 

### LINKED ISSUE #964: The successful/ unsuccessful message bar is not visible when Mozilla VPN promotional banner is displayed
**Build:**
Stage

**Affected platforms:**
Windows 10 64-bit
macOS 10.14
Android 11

**Browsers tested on:**
Firefox

**Steps to reproduce:**

1. Launch Firefox with a new profile.
2. Navigate to https://stage.fxprivaterelay.nonprod.cloudops.mozgcp.net and log in using an existing account.
3. Observe that the blue message bar "Successfully signed in as..." is not visible.
4. Close the Mozilla VPN promotional banner.
5. Observe that the blue message bar is visible.

**Actual result:**
The message bar is not visible when Mozilla VPN promotional banner is displayed.

**Expected result:** 
The message bar is visible when Mozilla VPN promotional banner is displayed.

successful msg not visible

### PR DESCRIPTION
**Pull request by maxxcrawford**
This PR solves the issue with the different levels of padding to make the content below the header appear correctly. 

By switching to `position: sticky`, it solves this!

### CODE REVIEW

File: static/scss/partials/main.scss [hunk 1]
```diff
@@ -1566,14 +1572,20 @@ input.input-has-error {
   visibility: visible;
   pointer-events: all;
   transition: all 0.3s ease;
+  max-height: 300px;
```
Comment (maxxcrawford) [inline static/scss/partials/main.scss:1575]: Magic Number™ – If there's a better way to animate open/shut, LMK.

File: static/js/app.js [hunk 2]
```diff
@@ -206,13 +206,6 @@ function toggleAliasCardDetailsVisibility(aliasCard) {
   detailsWrapper.classList.toggle("is-visible");
 }
 
-function resetBodyPadding() {
-  const header = document.querySelector("header");
-  const headerHeight = header.clientHeight;
-  document.body.style.paddingTop = headerHeight + "px";
-  return;
-}
-
```
Comment (maxxcrawford) [inline static/js/app.js:215]: No longer needed!

**Comment by maxxcrawford** [comment]
Note – I have temporarily deployed this to the [dev review site](https://dev.fxprivaterelay.nonprod.cloudops.mozgcp.net/).

File: static/scss/partials/main.scss [hunk 3]
```diff
@@ -55,7 +55,7 @@ body {
   display: flex;
   flex-direction: column;
   margin: 0;
-  padding: 83px 0 0 0;
+  padding: 0 0 0 0;
```
Comment (codemist) [inline static/scss/partials/main.scss:58]: Just wondering if we need this line if all the values are set to zero?

Comment (maxxcrawford) [reply static/scss/partials/main.scss:58]: I'll check – if anything, it might be a `padding: 0` only.

Comment (maxxcrawford) [reply static/scss/partials/main.scss]: Fixed!

**Review by codemist — APPROVED** [review]
Looks good! 👍

**Pull request merged**